### PR TITLE
Adding the Purify filter to the values that can contain simple HTML formatting

### PR DESCRIPTION
**Pull request by escopecz**
…ormatting

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [Y]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

I noticed by browsing through the UI that some places show HTML markup. Like:

Screenshot 2023-06-28 at 16 51 17

And in several places where the description can contain HTML markup. This PR is adding the `|purify` twig filter for those places so the basic HTML markup shows up and potential XSS attack is disabled.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Try to install a plugin via Marketplace UI (you'll have to approve that you've installed Mautic via Composer to get that option). Check that the modal box has well-formatted text.
3. Create a segment with a description that contain bold text. Check that the description renders correctly on the detail page as well as on the list page.
4. Do the same for Marketing Message description, email description and webhook description

### CODE REVIEW

File: app/bundles/PluginBundle/Resources/views/Integration/info.html.twig [hunk 1]
```diff
@@ -16,7 +16,7 @@
 {% if bundle.hasSecondaryDescription %}
 <div class="row mt-lg">
     <div class="col-xs-12">
-        {{ bundle.secondaryDescription|raw }}
```
Comment (mabumusa1) [inline app/bundles/PluginBundle/Resources/views/Integration/info.html.twig:19]: @escopecz I think it is a good idea to check where we have `raw` in the whole code. it is a potential bug

**Review by mabumusa1 — DISMISSED** [review]
Changes looks good

File: app/bundles/PluginBundle/Resources/views/Integration/info.html.twig (hunk 1 above)
Comment (escopecz) [reply app/bundles/PluginBundle/Resources/views/Integration/info.html.twig:19]: Good point. I went through the usages of `|raw` and found some that can be purifyied in https://github.com/mautic/mautic/pull/12528/commits/6a92d8675d9190c46c0efef69f7b36698bab8141

File: app/bundles/CoreBundle/Resources/views/Helper/builder.html.twig [hunk 2]
```diff
@@ -54,7 +54,7 @@
 
         <div class="code-editor {% if not isCodeMode %}hide{% endif %}">
             <div id="customHtmlContainer"></div>
-            <i class="text-muted">{{ 'mautic.core.code.mode.token.dropdown.hint'|trans|raw }}</i>
+            <i class="text-muted">{{ 'mautic.core.code.mode.token.dropdown.hint'|trans|puirfy }}</i>
```
Comment (kuzmany) [inline app/bundles/CoreBundle/Resources/views/Helper/builder.html.twig]: Typo

Comment (escopecz) [reply app/bundles/CoreBundle/Resources/views/Helper/builder.html.twig]: Thanks! Fixed

**Review by kuzmany — APPROVED** [review]

**Pull request merged**